src/main/java/com/rapid7/net/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.rapid7.net;
+
+final class Utils {
+  // private constructor
+  private Utils (){}
+  
+  /**
+   * Check if the contained string is <code>null</code> or empty. Trimming is NOT applied.
+   * @param s The String to be checked.
+   * @return <code>true</code> if the passed String is <code>null</code> or empty.
+   */
+  public static boolean isNullOrEmpty (String s){
```

Comment:
Instead of implementing our own, would it be better to use Apache Commons string utils? https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isEmpty(java.lang.String)